packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/di/app_dependencies.dart

```diff
@@ -87,13 +86,20 @@ class AppDependencies {
       ];
 
   List<Provider> get _httpClients => [
-        Provider<Dio>(
+        Provider<PlainHttpClient>(
+          create: (context) {
+            final client = PlainHttpClient();
```

Comment:
Considering this client is used for authentication I believe the base url should be configured as part of the template.